httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/AuthChallengeParser.java

```diff
@@ -116,6 +116,7 @@ public String toString() {
                     params.clear();
                 }
             }
+            if (token68 == null) token68 = "";
```

Comment:
@careylam could you please explain why this change is necessary? Please also note there are test cases failing due to this change. 
I also suggest this check be moved from the `AuthChallengeParser` to the `GGSSchemeBase` class.

Reply:
This line is to avoid GGSSchemeBase throwing MalformedChanllengeException. The token is not in the negociate reponse header "WWW-Authenticate: Negotiate".

I suggest to remove the null check on GGSSchemeBase. What do you think?

Reply:
I see no point in converting a null into an empty string. Where is the benefit? The token hasn't been provided. Period.